test/unit/rules/lint-attribute-indentation-test.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'attribute-indentation',
+
+  config: true,
+
+  good: [
+    //Non Block form with no params
+    '{{contact-details}}',
+    //Default config with open-invocation(< 80 chars)
+    //positional params
+    '{{contact-details firstName lastName}}',
+    //named params
+    '{{contact-details firstName=firstName lastName=lastName}}',
+    //Non Block form more than the default config characters (> 80 chars)
+    {
+      config: {
+        'open-invocation-max-len': 120
+      },
+      template: '{{contact-details firstName=firstName lastName=lastName avatarUrl=avatarUrl age=age address=address phoneNo=phoneNo}}'
+    },
+    //Open-invocation with multiple lines.
+    '{{contact-details' + '\n' +
+    '  firstName=firstName' + '\n' +
+    '  lastName=lastName' + '\n' +
+    '}}',
```

Comment:
Will attribute-indentation support not having to have }} on its own line. I like to indent over multiple lines, but retain the }} at the end

Reply:
We don't have an option to specify this style of indentation. Perhaps, we could create a sub rule?

cc: @rwjblue

Reply:
Worth a though, as I think this should be supported to:
```handlebars
{{my-component
  foo='bar'
  baz='qux'
  my-attr=(component 'my-other-component'
    foo='bar'
    baz='qux')}}
```

Reply:
for single line invocation, closing at the end of the last line seems ok (and we can update this rule to allow it).

@amk221 - Can you create an issue to track this?